adapters/repos/db/inverted/searcher.go

```diff
@@ -368,20 +374,26 @@ func (fs *Searcher) extractTokenizableProp(propName string, dt schema.DataType,
 		return nil, fmt.Errorf("expected value type to be string or text, got %v", dt)
 	}
 
-	propValuePairs := make([]*propValuePair, len(parts))
-	for i, part := range parts {
-		propValuePairs[i] = &propValuePair{
+	propValuePairs := make([]*propValuePair, 0, len(parts))
+	for _, part := range parts {
+		if sd.IsStopword(part) {
+			continue
+		}
+		propValuePairs = append(propValuePairs, &propValuePair{
 			value:        []byte(part),
 			hasFrequency: true,
 			prop:         propName,
 			operator:     operator,
-		}
+		})
 	}
 
-	if len(parts) > 1 {
+	if len(propValuePairs) > 1 {
 		return &propValuePair{operator: filters.OperatorAnd, children: propValuePairs}, nil
 	}
-	return propValuePairs[0], nil
+	if len(propValuePairs) == 1 {
+		return propValuePairs[0], nil
+	}
+	return nil, errors.Errorf("Invalid search term, only stopwords provided. Stopwords can be configured in Class.InvertedIndexCondig.Stopwords")
```

Comment:
Two small comments on the error message:

1. The convention in Go is to [start the message lowercased and not end in punctuation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).
2. Since the user will most likely interact with this configuration through JSON, I think it would be cleaner to use `class.invertedIndexCondig.stopwords` as that's most likely how the user would see their schema config.